src/game/default/g_cmd.c

```diff
@@ -757,11 +786,16 @@ _Bool G_AddClientToTeam(g_entity_t *ent, const char *team_name) {
 	if (ent->client->locals.persistent.team == team)
 		return false;
 
+	if (g_level.gameplay == GAME_DUEL && G_TeamSize(team) > 0) {
+		gi.ClientPrint(ent, PRINT_HIGH, "Only 1 player per team allowed in Duel mode\n");
+		return false;
+	}
+	
```

Comment:
I really appreciate the effort you've gone through to sanitize all of the requisite gameplay vars to enable duel mode. I'm slightly concerned that users might find it too tedious / difficult to setup? For this scenario, it might be better to force all existing clients to no team affiliation when gameplay is changed. Otherwise, enabling duel on a populated server would require full cooperation from everyone connected. What do you think?